Ajout d'images aux articles

apuyou opened the pull request:
Permet d'uploader une image et de préciser l'id d'une image quand on ajoute/édite un article.

apuyou committed:
Possibilité de récupérer le state de la class Image

apuyou committed:
Méthode pour uploader des images

apuyou committed:
Permettre l'ajout d'un id d'image à un article

trecouvr commented:
Pour le edit image, si `empty(intval($image))` il faudrait mieux garder l'image actuelle plutot que de passer a `null`. L'utilisateur ne va pas uploader son image a chaque fois qu'il veut editer un article.

apuyou commented:
En fait j'ai écrit le fix, mais ça ne me plaît pas, car il faut pouvoir supprimer l'image aussi (et j'ai pas envie d'écrire une méthode rien que pour ça).

Du coup ça va être à scoobydoo de gérer ça : si l'image n'est pas modifiée, il fait l'appel avec l'id de l'image actuelle.

apuyou committed:
Pouvoir garder l'image lors de l'édition d'un article

Pour le paramètre $image : -1 pour supprimer, 0 pour conserver, nouvel id pour changer

apuyou committed:
Ajout de paramètres par défaut pour $image
